packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -40,6 +61,167 @@ abstract class BindingBase {
     assert(() { _debugInitialized = true; return true; });
   }
 
+  /// Called, in checked mode, to register service extensions.
```

Comment:
Yeah, this requires the profile mode though. Release builds shouldn't have any service extensions, that's a security vulnerability.

I can turn on the service extensions temporarily until we have a solution for profile builds if you want.